css/pages/faq.css

```diff
@@ -51,6 +51,10 @@ h1 {
   right: 1rem;
 }
 
+.user_FAQ{
```

Comment:
Can we have the name of class as `faq_link` or something more close to what the class is meant for? 🤔

Reply:
why not! :)